javascript-loops-practice/add-suffix-to-all.js

```diff
@@ -0,0 +1,8 @@
+/* exported addSuffixToAll */
+function addSuffixToAll(words, suffix) {
+  var arr = [];
+  for (var i = 0; i < words.length; i++) {
+    arr.push(words[i] += suffix);
```

Comment:
You shouldn't use `+=` here, since that is modifying the values in the original array. You can just use `+`